Sources/ArcGISToolkit/Components/OfflineMapAreasView/OfflineMapAreasView.MapViewModel.swift

```diff
@@ -30,16 +30,9 @@ extension OfflineMapAreasView {
         /// The preplanned offline map information.
         @Published private(set) var preplannedMapModels: Result<[PreplannedMapModel], Error>?
         
-        /// A Boolean value indicating whether the map has preplanned map areas.
-        @Published private(set) var hasPreplannedMapAreas = false
-        
-        /// A Boolean value indicating whether the user has authorized notifications to be shown.
-        var canShowNotifications = false
-        
         init(map: Map) {
             offlineMapTask = OfflineMapTask(onlineMap: map)
             portalItemID = map.item?.id?.rawValue
-            JobManager.shared.resumeAllPausedJobs()
```

Comment:
This isn't needed here. The model/view should start the individual job. That's also better in case you have a UX that shows the job is paused, then moving this to the preplanned model (which is what I've done in this PR), makes for an easier transition and gives more flexibility in the future.